### ISSUE TITLE
ignore bar to use tile or switch-grid

### ISSUE THREAD

**Issue by gitlerat** [wmutils/core#66]
Hi,
i am currently trying wmutils and have some problems...

I am using tint2 and if I start the tile.sh script it tiles the bar too. Using "ignw -s tint2" does not do anything.
Also if a start switch-grid.sh the following error appears:
```
switch_grid.sh: 32: switch_grid.sh: bc: not found
switch_grid.sh: 52: test: -eq: argument expected
switch_grid.sh: 61: switch_grid.sh: arithmetic expression: division by zero: " SH/ROWS - GAP - BW "
```
I am glad about every help :)

**Comment by dcat** [comment]
first of all, `ignw` takes window ID’s as arguments, not window names. 


> switch_grid.sh: 32: switch_grid.sh: bc: not found

tells you that you don’t have `bc` installed.

**Comment by lwilletts** [comment]
There might be an option in tint2's documentation to set it to 'background' or something similar. If you set this you shouldn't have to use `ignw`

**Comment by z3bra** [comment]
From the [tint2 wiki][0]: 

> Window managers that do not follow exactly the EWMH specification might not interact well with tint2 (issue 627).

wmutils doesn't follow the EWMH specification _at all_. Which means that tint2 will not play well with wmutils toolset. In your case, it means that the windows created by tint2 do not set the `override_redirect` attribute (which wmutils uses to ignore windows), and rely instead on the EMWH atoms to define it's windows as "docks".

You might try your luck using `atomx`, which has the ability to retrieve an set any atom (not just EWMH ones), but at this point you'll be re-implementing a full EWMH window manager in a scripting language, and perfs will be deceiving.

TL;DR: Tint2 expect an EWMH compliant window manager to run, which is not what wmutils is. They are then not compatible.

[0]: https://gitlab.com/o9000/tint2
